src/applications/vaos/new-appointment/redux/actions.js

```diff
@@ -1059,7 +1059,7 @@ export function submitAppointmentOrRequest(history) {
           ...additionalEventData,
         });
         resetDataLayer();
-        history.push('/new-appointment/confirmation');
+        history.push(`/requests/${requestData.id}/true`);
```

Comment:
We should only redirect to the details page when the homepage refresh flag is true